### PR TITLE
practice spawns

### PR DESCRIPTION
**Pull request by evilantAU**

### CODE REVIEW

**Comment by evilantAU** [comment]
practice spawns

File: ssqc/client.qc [hunk 1]
```diff
@@ -1518,7 +1524,10 @@ void () ClientKill = {
     TeamFortress_RemoveTimers();
 
     // players can't suicide again for 10 seconds
-    self.suicide_time = time + 5 + random() * 5;
+    if(!allowpracspawns)
+	self.suicide_time = time + 5 + random() * 5;
+     else 
+	self.suicide_time = time;
```
Comment (drzel) [inline ssqc/client.qc:1530]: indent seems broken here

File: ssqc/client.qc [hunk 2]
```diff
@@ -1671,6 +1680,15 @@ void (entity e) ValidateUser = {
 
 entity () SelectSpawnPoint =
 {
+    if(allowpracspawns && self != world)
+    {
+	local entity spot2;
+	spot2 = TeamFortress_GetPracticeSpawn(self);
+	
+	if(spot2 != world)
+		return spot2;
```
Comment (drzel) [inline ssqc/client.qc:1689]: also here should be across one

**Review by drzel — CHANGES_REQUESTED** [review]
It's mostly fine, and I like the feature a lot, a few small requests:

- Instead of suicide time penalties removed, can we just have another localinfo to set the penalty length?
- Indents are all out, there's a few places where it's actually reversed, could you fix that up?
- There's a merge conflict with the readme that needs to be fixed.

File: ssqc/client.qc (hunk 1 above)
Comment (lordee) [reply ssqc/client.qc]: What's the thinking for yet another setting for suicide time for practice spawns? Seems like a lot of bloat and configuration for no reason

File: ssqc/client.qc (hunk 2 above)
Comment (lordee) [reply ssqc/client.qc:1689]: Can't see the issue here?

Comment (lordee) [reply ssqc/client.qc:1689]: I can in desktop mode actually... yeah needs an indent

**Pull request merged**